app/services/newspaper_works/pluggable_derivative_service.rb

```diff
@@ -54,17 +54,64 @@ def respond_to_missing?(method_name)
     self.class.allowed_methods.include?(method_name) || super
   end
 
+  # get derivative services relevant to method name and file_set context
+  #   -- omits plugins if particular destination exists or will soon.
+  def services(method_name)
+    result = plugins.map { |plugin| plugin.new(file_set) }.select(&:valid?)
+    result.select do |plugin|
+      dest = nil
+      dest = plugin.class.target_ext if plugin.class.respond_to?(:target_ext)
+      !skip_destination?(method_name, dest)
+    end
+  end
+
   def method_missing(name, *args, **opts, &block)
     if respond_to_missing?(name)
       # we have an allowed method, construct services and include all valid
       #   services for the file_set
-      services = plugins.map { |plugin| plugin.new(file_set) }.select(&:valid?)
+      # services = plugins.map { |plugin| plugin.new(file_set) }.select(&:valid?)
       # run all valid services, in order:
-      services.each do |plugin|
+      services(name).each do |plugin|
+        # dest = nil
```

Comment:
Remove commented-out code?

Reply:
Will do this in same commit as README update, per your feedback below.